index.js

```diff
@@ -20,14 +20,20 @@ module.exports = fp(function from (fastify, opts, next) {
     'application/json',
     ...(opts.contentTypesToEncode || [])
   ])
+
+  const retryMethods = new Set(opts.retryMethods || [
+    'GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'
```

Comment:
PUT and DELETE should not be retriable.

Reply:
will remove them, although MDN says they are indepotent